src/main/java/frc/robot/commands/ArmToGoalCommand.java

```diff
@@ -55,6 +61,9 @@ public void initialize() {
 
     @Override
     public void execute() {
+        if((flags & IGNORE_POSITION) == IGNORE_POSITION) {
```

Comment:
I would actually argue for the reinstatement of WristAngleCommand - but in a very specific way. I'll commit what I'm thinking of, and then this flag can be removed. The reason I don't like this flag is because it completely nullifies a part of the command, and at this point you may as well write a more concise command.